src/app.js

```diff
@@ -1 +1,35 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { userTerminal } = require('./terminal');
+const { randomNumber } = require('./randomNumber');
+const { checkResult } = require('./checkResult');
+const random = randomNumber();
+let questionMessage = 'Write 4 different numbers   ';
```

Comment:
Why you need whitespaces in the end of string?
```suggestion
let questionMessage = 'Write 4 different numbers';
```